bricks/dart_frog_new/__brick__/{{filename}}

```diff
@@ -1,3 +1,6 @@
 import 'package:dart_frog/dart_frog.dart';
```

Comment:
Shall we include the space between the import and the first line to have a correctly formatted file?

```suggestion
import 'package:dart_frog/dart_frog.dart';

```

Reply:
This creates an unnecessary space, reverting it in the next pr